index.js

```diff
@@ -21,7 +21,7 @@ function addValueBindLocator() {
 
 function loadAndWaitForAureliaPage(pageUrl) {
   function onAureliaComposed(onReady) {
-    if (!!document.querySelector("[aurelia-app]")) {
+    if (!!document.querySelector("[aurelia-app]").aurelia) {
```

Comment:
This is not type-safe and I think it can be problematic in case there is no element matching `[aurelia-app]`. You will get:

```js
Uncaught TypeError: Cannot read property 'aurelia' of null
```

Would be safer to do: 
```js
var aureliaElement = document.querySelector("[aurelia-app]")
if (aureliaElement && !!aureliaElement.aurelia) {
```